lib/widgets/messages/message_input_widget.dart

```diff
@@ -48,48 +49,60 @@ class MessageInputWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     final Messages messageData = Provider.of<Messages>(context);
     final User user = Provider.of<Users>(context).currentUser;
-
     return Container(
-      padding: EdgeInsets.all(5),
+      height: 60,
+      margin: EdgeInsets.all(10),
+      padding: EdgeInsets.only(left: 10),
+      decoration: BoxDecoration(
+          color: Theme.of(context).primaryColor,
+          borderRadius: BorderRadius.circular(10),
+          boxShadow: [
+            BoxShadow(
+              color: Colors.black,
+              offset: Offset(0.0, 3.0),
+              blurRadius: 5.0,
+            ),
+          ]),
       child: Row(
         crossAxisAlignment: CrossAxisAlignment.center,
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
         children: [
-          Container(
-            padding: EdgeInsets.symmetric(horizontal: 5),
-            width: MediaQuery.of(context).size.width - 100,
-            decoration: BoxDecoration(
-              color: Theme.of(context).primaryColor,
-              borderRadius: BorderRadius.circular(5),
-            ),
-            child: TextField(
-              style: Theme.of(context).textTheme.body1,
-              textCapitalization: TextCapitalization.sentences,
-              decoration: InputDecoration(
-                hintStyle: Theme.of(context).textTheme.body1,
-                hasFloatingPlaceholder: true,
-                hintText: 'Say something nice...',
+          Expanded(
+            flex: 3,
+            child: Container(
+              padding: EdgeInsets.all(5),
+              child: TextField(
+                style: Theme.of(context).textTheme.body1,
+                textCapitalization: TextCapitalization.sentences,
+                decoration: InputDecoration(
+                  hintStyle: Theme.of(context).textTheme.body1,
+                  hasFloatingPlaceholder: true,
+                  hintText: 'Say something nice...',
```

Comment:
Did we want to move any hard coded strings like this to a central file and pull them in as variables? That way we can change it in one place and the text is changed anywhere it is used. I believe it is also best practice in mobile as it can help support different languages.

Reply:
AFAIK its best practice in general I'll update with a constants file or similar for strings and date formats